ietf/group/templatetags/group_filters.py

```diff
@@ -19,14 +19,13 @@ def active_nomcoms(user):
     if not (user and hasattr(user, "is_authenticated") and user.is_authenticated):
         return []
 
-    groups = []
-
-    groups.extend(Group.objects.filter(
-        role__person__user=user,
-        type_id='nomcom',
-        state__slug='active').distinct().select_related("type"))
-
-    return groups
+    return list(
+        NomCom.objects.filter(
+            group__role__person__user=user,
+            group__type_id='nomcom',  # just in case...
+            group__state__slug='active',
+        )
```

Comment:
probably want distinct() here - several people end up with multiple roles in a NomCom.group.

Reply:
also consider an order_by, perhaps on `group__acronym` (but maybe making the implicit `pk` ordering explicit is enough.

Reply:
Where was the `select_related("type")` used in the existing code?

Reply:
(looks like it _wasn't_. Some other code must have gone away...)

Reply:
That's right, it wasn't being used. Will add the `distinct()` and an ordering